packages/jest-environment-jsdom/src/index.js

```diff
@@ -22,6 +22,7 @@ class JSDOMEnvironment {
   moduleMocker: ?ModuleMocker;
 
   constructor(config: ProjectConfig): void {
+    const jsdomInitialized = process.hrtime();
```

Comment:
I'd prefer to have it beneath the `JSDom.jsdom` call, but that makes stuff fail with an odd flow error...